### PR TITLE
Feature/increase output UI size

### LINKED ISSUE #974: Increase UI component size for accesibility
Based on user feedback and from some tests with a new accessibility device, we would like to increase the size and/or "clickable area" for specific components in the UI. 

We are testing an eye tracker and the precision is not enough to use the current UI size. 

**What have we tested?**
We already tried increasing the size of the UI component from:
Display -> Increase UI size
Display -> Increase Font size

We suggest that this size should be at least 66% of the size of a tile. (Only for this accessibility device, but we suggest to increase it for all users, especially the ones with motor impairments).

Buttons 1 and 2 are crucial for usability. 
Captura de Pantalla 2021-07-13 a la(s) 14 52 22

### PR DESCRIPTION
**Pull request by tomivm**
related to #974

### CODE REVIEW

File: src/components/Board/Output/SymbolOutput/SymbolOutput.css [hunk 1]
```diff
@@ -54,6 +54,73 @@
...
+  height: 100%;
+  justify-content: center;
+  align-items: center;
+}
+
 .SymbolOutput__right__btns .MuiFormControlLabel-label {
   font-size: 11px;
 }
+
+.Output__button__sm {
```
Comment (RodriSanchez1) [inline src/components/Board/Output/SymbolOutput/SymbolOutput.css]: add the button tag before .Output_button__sm like the others classes

File: src/components/Board/Output/SymbolOutput/ClearButton/ClearButton.js [hunk 2]
```diff
@@ -27,16 +15,26 @@ export class ClearButton extends Component {
...
+            }
+          />
         </IconButton>
       </Scannable>
     );
   }
 }
 
-export default withStyles(styles)(ClearButton);
+export default withStyles(null)(ClearButton);
```
Comment (RodriSanchez1) [inline src/components/Board/Output/SymbolOutput/ClearButton/ClearButton.js:40]: You can export directly the componente without the withStyles, like `export default (ClearButton)` and also deleate the import.

Comment (RodriSanchez1) [inline src/components/Board/Output/SymbolOutput/ClearButton/ClearButton.js]: You can export directly the componente without the withStyles, like `export default (ClearButton)` and also deleate the import.

**Review by RodriSanchez1 — CHANGES_REQUESTED** [review]

File: src/components/Board/Output/SymbolOutput/ClearButton/ClearButton.js [hunk 3]
```diff
@@ -27,16 +15,26 @@ export class ClearButton extends Component {
   };
 
   render() {
-    const { classes, hidden, ...other } = this.props;
+    const { classes, hidden, increaseOutputButtons, ...other } = this.props;
```
Comment (RodriSanchez1) [inline src/components/Board/Output/SymbolOutput/ClearButton/ClearButton.js]: You can delete the prop classes

File: src/components/Board/Output/SymbolOutput/SymbolOutput.css (hunk 1 above)
Comment (tomivm) [reply src/components/Board/Output/SymbolOutput/SymbolOutput.css]: why?

**Review by martinbedouret — CHANGES_REQUESTED** [review]
` Warning: A component is changing an uncontrolled input to be controlled. This is likely caused by the value changing from undefined to a defined value, which should not happen. Decide between using a controlled or uncontrolled input element for the lifetime of the component. More info: https://reactjs.org/link/controlled-components
`

**Review by martinbedouret — APPROVED** [review]

**Pull request merged**